client/cloud_credentials_project_assignment_test.go

```diff
@@ -2,6 +2,7 @@ package client_test
 
 import (
 	"errors"
+
 	. "github.com/env0/terraform-provider-env0/client"
 	"github.com/golang/mock/gomock"
 	. "github.com/onsi/ginkgo"
```

Comment:
@sagilaufer1992 - it's possible (guessing here) that it's because there are two packages in the same directory (client and client_test).

About ginko it's okay when it's an external library.
See here: https://staticcheck.io/docs/checks#ST1001